### PR TITLE
Initial static pages

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: packages/publisher-dapp/public/index.html [hunk 1]
```diff
@@ -24,6 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <!-- Muli Font -->
+    <link href="https://fonts.googleapis.com/css?family=Muli:200,200i,300,300i,400,400i,600,600i,700,700i,800,800i,900,900i&display=swap" rel="stylesheet">
```
Comment (ksridharbabuus) [inline packages/publisher-dapp/public/index.html]: Do we have the Fall back fonts? Image if we are accessing the site from a location where we dont have access to the Google Fonts, need to have a fall back fonts. Please make sure that we have one.

File: packages/publisher-dapp/src/Pages/Overview/content.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	 	}
+	}
+];
+
+export const ProgramMemberShipDetailsList = [
+	{
+		title: "Program Membership Details",
+		description: "Lorem ipsum dolor sit amet, mei te indoctum convenire. Sumo urbanitas moderatius eam ut, pro ex prima harum ri de ns. Vis obli que nusquam te, cum ex minim molestie. Eam ex modus luptatum adipiscing, pri id patrioque instructior, mel id tale definitionem.",
+	 	list: [
+			"AI services dashboard",
```
Comment (ksridharbabuus) [inline packages/publisher-dapp/src/Pages/Overview/content.js]: Shall we look at the Content Models as the same lines as Contentful offerings.
Cotentful (https://www.contentful.com) is just a reference for designing the model.

File: packages/publisher-dapp/src/HOC/withLightHeaderAndFooter.js [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
+import Container from "@material-ui/core/Container";
+import Box from "@material-ui/core/Box";
+
 const withLightHeaderAndFooter = Component => {
   return props => (
     <Fragment>
-      <header>Light Header</header>
-      <Component {...props} />
-      <Footer />
+      <Header />
```
Comment (ksridharbabuus) [inline packages/publisher-dapp/src/HOC/withLightHeaderAndFooter.js]: Not able to locate the Document Header content like Title and Meta tags which is very much required for the better SEO.
Let us look at the package like react-helmet to have the generic Doc Head section aligned to the SEO optimization.

File: packages/publisher-dapp/src/Pages/Overview/content.js (hunk 2 above)
Comment (Vivek205) [reply packages/publisher-dapp/src/Pages/Overview/content.js]: Will explore further and track it here https://github.com/singnet/snet-dapp-monorepo/issues/7

File: packages/publisher-dapp/src/HOC/withLightHeaderAndFooter.js (hunk 3 above)
Comment (Vivek205) [reply packages/publisher-dapp/src/HOC/withLightHeaderAndFooter.js]: Will track it here https://github.com/singnet/snet-dapp-monorepo/issues/8 and raise a separate PR for this

File: packages/publisher-dapp/public/index.html (hunk 1 above)
Comment (Vivek205) [reply packages/publisher-dapp/public/index.html]: Will connect with Sujith for the fallback fonts ( which one to use ) and will add it here. https://github.com/singnet/snet-dapp-monorepo/issues/9

**Pull request merged**